python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -306,9 +348,6 @@ def handle_failure(self):
         raise RuntimeError("Worker crashed during training. "
                            "Training unsuccessful.")
 
-    def get_latest_checkpoint(self) -> Optional[Dict]:
-        return self.latest_checkpoint
```

Comment:
should we make `latest_checkpoint` a property as well? That way it's clear that it's meant to be directly accessed by upstream.

Reply:
We can make it a property if any additional logic is needed.

https://www.python.org/dev/peps/pep-0008/
```
For simple public data attributes, it is best to expose just the attribute name, without complicated accessor/mutator methods. Keep in mind that Python provides an easy path to future enhancement, should you find that a simple data attribute needs to grow functional behavior. In that case, use properties to hide functional implementation behind simple data attribute access syntax.
```